playbooks/robusta_playbooks/java_pod_troubleshooting.py

```diff
@@ -0,0 +1,150 @@
+from robusta.api import *
+from robusta.integrations.kubernetes.process_utils import ProcessFinder, ProcessType
+from typing import List
+import traceback
+
+from robusta.integrations.kubernetes.custom_models import JAVA_DEBUGGER_IMAGE
+
+@action
+def java_debugger(event: PodEvent, params: ProcessParams):
```

Comment:
I'm not crazy about calling this `java_debugger` because that implies we attach an actual debugger here which we don't. (This is what `python_debugger` does so it's confusing if `java_debugger` doesn't.)

Alternative names could be:
* java_ps
* java_inspector
* <your idea here>

Whatever the action is called, it is most similar to the playbook action currently called `advanced_debugging_options`. I suggest we give them the same naming structure. (E.g. rename `advanced_debugging_options` to `python_ps` or `python_inspector`.) 

I'm not just nit-picking here. The consistency is important for making these playbooks self explanatory and easy to use for end users. So please update all the messages in both playbooks according to the names you choose. (E.g. "Java debugging session on pod" -> "Java Inspector for pod...")

Reply:
im with you

Reply:
java_process_inspector? python_process_inspector?

Reply:
I think java_inspector is good. We can leave out the word precess.